libdnf/dnf-sack.cpp

```diff
@@ -2237,21 +2267,50 @@ static void setModuleExcludes(DnfSack *sack, const std::vector<std::string> &hot
 
     dnf_sack_set_module_excludes(sack, excludeQuery.getResultPset());
     dnf_sack_add_module_excludes(sack, excludeProvidesQuery.getResultPset());
+    dnf_sack_set_module_includes(sack, includeQuery.getResultPset());
 }
 
 }
 
-void dnf_sack_filter_modules(DnfSack *sack, GPtrArray *repos, const char *install_root,
-    const char * platformModule)
+std::vector<std::shared_ptr<ModulePackage>> requiresModuleEnablement(DnfSack * sack, const libdnf::PackageSet * installSet)
 {
+    DnfSackPrivate *priv = GET_PRIVATE(sack);
+    if (!priv->module_includes || !priv->moduleContainer) {
+        return {};
+    }
+    libdnf::PackageSet tmp(sack);
+    tmp += *const_cast<libdnf::PackageSet*>(installSet);
+    tmp /= priv->module_includes;
+
+    if (tmp.empty()) {
+        return {};
+    }
+    auto toEnable = priv->moduleContainer->requiresModuleEnablement(*installSet);
+    return toEnable;
+}
+
+void dnf_sack_filter_modules(DnfSack * sack, DnfModulePackageContainer * moduleContainer,
```

Comment:
Can you rename this and keep the original function for compatibility with PackageKit?